bleak/backends/bluezdbus/scanner.py

```diff
@@ -143,6 +144,42 @@ async def start(self):
 
         await self.resume()
 
+    async def resume(self):
```

Comment:
This does not really match the commit description?

Reply:
I think it does though, it was added in [this commit ](https://github.com/workaroundgmbh/bleak/pull/33/commits/61c12e54b5750bae751d78889c1bfb0d5d953497#diff-cc469ec95c55bf20bb32d39bf09be9cec2dfc6820e40e857367f8b49915cddcdR144)